packages/react-storage/package.json

```diff
@@ -66,7 +66,8 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "15.2 kB"
+      "limit": "15.2 kB",
+      "ignore": ["@aws-amplify/storage"]
```

Comment:
Ignore this scoped package as it is already taken into account via the umbrella package `aws-amplify`